api/apiFunctions.test.tsx

```diff
@@ -1,317 +1,493 @@
+          $id: '000',
+        },
+      ];
+
+      apiFunctions.getCurrentUserEntries.mockResolvedValue(userEntries);
+
+      const result = await apiFunctions.getCurrentUserEntries();
+
+      for (const entry of result) {
+        expect(entry.eliminated).toEqual(true);
+      }
+    });
 
-    const result = await apiFunctions.getAllLeagues();
+    it('should get all user entries and check if eliminated is equal to false', async () => {
+      const userEntries = [
+        {
+          eliminated: false,
+          league: {},
+          name: 'name',
+          selectedTeams: [],
+          user: '000',
+          $id: '000',
+        },
+      ];
+
+      apiFunctions.getCurrentUserEntries.mockResolvedValue(userEntries);
+
+      const result = await apiFunctions.getCurrentUserEntries();
+
+      for (const entry of result) {
+        expect(entry.eliminated).toEqual(false);
+      }
+    });
 
-    expect(result).toEqual([
-      {
-        leagueId: '1',
-        leagueName: 'League One',
-        logo: '',
-        participants: ['user1', 'user2'],
-        survivors: ['user1'],
-      },
-      {
-        leagueId: '2',
-        leagueName: 'League Two',
-        logo: '',
-        participants: ['user3', 'user4'],
-        survivors: ['user4'],
-      },
-    ]);
-  });
-  it('should return error upon unsuccessful call', async () => {
-    apiFunctions.getAllLeagues.mockRejectedValue(
-      new Error('Error getting all leagues'),
-    );
-
-    await expect(apiFunctions.getAllLeagues()).rejects.toThrow(
-      'Error getting all leagues',
-    );
+    it('should get all user entries and check if eliminated is equal to null', async () => {
+      const userEntries = [
+        {
+          eliminated: null,
+          league: {},
+          name: 'name',
+          selectedTeams: [],
+          user: '000',
+          $id: '000',
+        },
+      ];
+
+      apiFunctions.getCurrentUserEntries.mockResolvedValue(userEntries);
+
+      const result = await apiFunctions.getCurrentUserEntries();
+
+      for (const entry of result) {
+        expect(entry.eliminated).toEqual(null);
+      }
+    });
   });
-});
 
-describe('get users id', () => {
-  it('should return user document ID', async () => {
-    const userId = '123';
-    const response = '456';
-    apiFunctions.getUserDocumentId.mockResolvedValue(response);
-    const result = await apiFunctions.getUserDocumentId(userId);
-    expect(result).toEqual(response);
-  });
-  it('should return error upon unsuccessful call', async () => {
-    apiFunctions.getUserDocumentId.mockRejectedValue(
-      new Error('Error getting user document ID'),
-    );
-    await expect(apiFunctions.getUserDocumentId('123')).rejects.toThrow(
-      'Error getting user document ID',
-    );
-  });
-});
+  xdescribe('get all leagues', () => {
+    it('should return all leagues upon successful call', async () => {
+      const mockAllLeagues = [
+        {
+          leagueId: '1',
```

Comment:
why is there all this empty space?

Reply:
@ryandotfurrer not sure what you are taking about .... I looked at the code and don't see any empty space.  I removed a few lines but that is about it.  Check now?